### DISCUSSION TITLE
CVE-2022-30778

### DISCUSSION THREAD

**Discussion by feltrine-lnrs** [laravel/framework#42998] [General]
I'm using the whitesource tool to scan the applications and a vulnerability related to [CVE-2022-30778](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-30778) was reported , is there any fix or discussion about that?

**Comment by henzeb** [comment]
Do you use the latest version of laravel (9.19.0 as of writing?)

**Reply by feltrine-lnrs** [reply]
Yes, I just updated the version trying to fix the problem.

**Comment by peternijssen** [comment]
More information here: https://github.com/1nhann/vulns/issues/1
